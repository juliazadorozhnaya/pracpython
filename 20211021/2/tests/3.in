:cos x y cos(x+y)
cos 1 2
:hello "Hello!!!"
hello
quit "{}:{}"