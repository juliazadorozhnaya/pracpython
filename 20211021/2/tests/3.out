-0.9899924966004454
Hello!!!
3:5
